Add methods for finding a font with the right size

codetent opened the pull request:
Needs to be checked with the mentioned Linux distribution if this solves the issue #10.

Akuli commented:
I have read your changes.

- Instead of a `tuple` conversion in `teek/_font.py`, it should return `to_tcl(self._font_description)`, where `to_tcl` is imported from `_tcl_calls.py`. This was a bug. It should have always called `to_tcl` on the font description before returning it from `to_tcl`, as is documented [here](https://teek.readthedocs.io/en/latest/tcl-calls.html#python-to-tcl-conversion):

> If the value has a `to_tcl()` method, it’s called with no arguments. It should return a string that will be passed to Tcl.

- Aren't `_check_font`, `_create_font` and `_create_named_font` a bit necessary? I think using `teek.Font.families()[0]` instead of `'Helvetica'` might be enough, because the `size` should behave nicely for a font that is actually installed on my system. This is true for at least every font on this system, as I have shown [here](https://github.com/Akuli/teek/issues/10#issuecomment-497072935).

codetent commented:
- I changed the tuple conversion to `to_tcl`
- I checked the size property for each font family installed on my system and discovered that the actual size can also vary if the font is available. Therefore, I replaced `'Helvetica'` with a function call which returns a font family whose size is equal to the given one.

Akuli commented:
Awesome :D
